htdocs/bom/class/bom.class.php

```diff
@@ -486,13 +486,13 @@ public function fetchLinesbytypeproduct($typeproduct = 0)
 	/**
 	 * Load list of objects in memory from the database.
 	 *
-	 * @param  string      		$sortorder    Sort Order
-	 * @param  string      		$sortfield    Sort field
-	 * @param  int         		$limit        Limit
-	 * @param  int         		$offset       Offset
-	 * @param  string   		$filter       Filter USF
-	 * @param  string      		$filtermode   Filter mode (AND or OR)
-	 * @return array|int        			  int <0 if KO, array of pages if OK
+	 * @param  string               $sortorder    Sort Order
+	 * @param  string               $sortfield    Sort field
+	 * @param  int                  $limit        Limit
+	 * @param  int                  $offset       Offset
+	 * @param  string               $filter       Filter USF
+	 * @param  string               $filtermode   Filter mode (AND or OR)
+	 * @return array|int                              int <0 if KO, array of pages if OK
```

Comment:
For static analysis, we need to specify the fields of the array return value.
I suspect it it the following
```
* @param int<-1,-1>|BOM[]
```

Reply:
This is not part of the PR, I am guessing that phpcs/phpcbf made this change to the spacing for some reason... tabs to spaces. But your analysis looks right.

Reply:
Sometime phpcbf is not doing the job as expected.
If you update it it could show other notices in other code leading to more changes to "fix" phan.